packages/compass-crud/src/stores/crud-store.js

```diff
@@ -201,6 +202,7 @@ const configureStore = (options = {}) => {
         isReadonly: false,
         isTimeSeries: false,
         status: DOCUMENTS_STATUS_INITIAL,
+        debouncing: false,
```

Comment:
This is not a very good variable name for what's going on,  but I can't think of anything better yet.